packages/vue/src/components/atoms/SfImage/SfImage.stories.js

```diff
@@ -87,7 +129,7 @@ storiesOf("Molecules|Image", module)
     {
       info: {
         summary: `<h2>Usage</h2>
-       <pre><code>import { SfImage } from "@storefront-ui/vue"</code></pre>
+       <pre><code>import { SfImage } from "@storefrontui/vue"</code></pre>
```

Comment:
```suggestion
       <pre><code>import { SfImage } from "@storefront-ui/vue"</code></pre>
```
because [@storefront-ui](https://www.npmjs.com/package/@storefront-ui/vue) is right